### ISSUE TITLE
Silencing "a jupyter kernel connection is required..." message

### ISSUE THREAD

**Issue by agahkarakuzu** [jupyter-book/thebe#776]
Hi! 

I was wondering if there is an option to suppress this message in an output rendered through MyST:

https://github.com/jupyter-book/thebe/blob/2f69ba2bf42a3bde8bbd28de7359f355f3c8781b/packages/core/src/utils.ts#L29

For a cleaner look, I want to add this information in the figure caption and suppress from the output (if possible).

Thank you!

**Comment by stevejpurves** [comment]
Hi @agahkarakuzu, one way to suppress the output is to add a placeholder to the figure, that should/would remove that message. And it will look nicer/cleaner in the process - https://mystmd.org/guide/reuse-jupyter-outputs#placeholder-content - does that fit your use case?

**Comment by agahkarakuzu** [comment]
Hi @stevejpurves, thanks for responding! I already have an interactive figure (widgets are modifying traces on it when connected to a kernel) rendered from the labeled cell output at the initial values of the sliders, and the warning appears right below it. So I assume using a placeholder would remove the content of interest?

**Comment by stevejpurves** [comment]
@agahkarakuzu it would replace the entire contents of the figure (all cell outputs) only on initial render, as soon as you connected to a kernel and ran the figure (cell/notebook) it would be replaced with your widgets.

It sounds like your cell might be producing 2 outputs though? the first an interactive graph that does not require a kernel and the second containing widgets?

**Comment by agahkarakuzu** [comment]
The widget callback is creating/updating the interactive figure.

This is how it looks on the page:

![Image](https://github.com/user-attachments/assets/c637079b-3843-4c71-b239-6d57b0a464db)

and on the notebook:

![Image](https://github.com/user-attachments/assets/d8fa5ed7-b333-46a5-8511-7357fa817dba)

with the following input: 

```python
#| label: interactCylinder

@interact(Ncyl=widgets.IntSlider(min=1, max=100, step=1, value=30,description="# cylinders"), R=widgets.FloatSlider(min=0.5, max=5, step=0.5, value=1.5, description="R cylinder"))
def update(Ncyl, R):
    """Update the visualization based on widget values."""
    visualize_random_cylinders(Ncyl=Ncyl, R=R, L=100)
```

**Comment by stevejpurves** [comment]
yes I thought so. I take it you would also want to remove the `interactive(children=...`?

so it looks like this line in the https://github.com/jupyter-book/myst-theme/blob/50c625864f091f52dc563cd3bd486e68de3625d1/packages/jupyter/src/jupyter.tsx#L90-L91 @myst-theme/jupyter package is the culprit but there's no configuration available for that right now, seems like it could be a "[theme option](https://github.com/jupyter-book/myst-theme/blob/50c625864f091f52dc563cd3bd486e68de3625d1/themes/article/template.yml#L15)" which would then need to be propagated through, to the PassiveOutputRenderer.

it might be better to wait on https://github.com/jupyter-book/myst-theme/pull/429 though.!?

**Comment by agahkarakuzu** [comment]
Thank you so much for the clarification!

I'll keep an eye on this PR as it is not really urgent and pretty glad to learn about rehydration through a PR :)

**Comment by stevejpurves** [comment]
great, closing this for now then
